dp-core/vr_flow.c

```
@@ -2241,17 +2241,23 @@ vr_flow_table_reset(struct vrouter *router)
     return;
 }
 
+void
+compute_size_oflow_table(int *oentries, int entries)
+{
+    /*
+    * Overflow entries is 20% of the main flow table
+    * adjusted to next 1k
+    */
+    if (!*oentries)
+        *oentries = ((entries / 5) + 1023) & ~1023;
+}
+
 static int
 vr_flow_table_init(struct vrouter *router)
 {
     if (!router->vr_flow_table) {
```

Maybe it should return an error if it is set, because that means a programmer error. Or at least an `ASSERT` would be helpful here.